benches/bench.rs

```diff
@@ -2,11 +2,32 @@
 extern crate bencher;
 extern crate crc32fast;
 extern crate rand;
+extern crate crc32_v2;
 
 use bencher::Bencher;
+use crc32_v2::crc32;
 use crc32fast::Hasher;
 use rand::Rng;
 
+fn bench_crc32_v2(b: &mut Bencher, size: usize) {
+    let mut bytes = vec![0u8; size];
+    rand::thread_rng().fill(&mut bytes[..]);
+
+    b.iter(|| {
+        crc32(bencher::black_box(0), bencher::black_box(&bytes));
```

Comment:
```suggestion
        bencher::black_box(crc32(bencher::black_box(0), bencher::black_box(&bytes)));
```

Reply:
Thanks for the catch, @srijs. Makes sense. Sorry, I was too excited at first and forgor to wrap it with `black_box`.